slurm/src/slurmcc/partition.py

```diff
@@ -45,11 +46,19 @@ def __init__(
         assert len(buckets) > 0
         self.max_scaleset_size = max_scaleset_size
         self.vcpu_count = buckets[0].vcpu_count
-        slurm_memory = buckets[0].resources.get("slurm_memory")
-        if slurm_memory:
-            self.memory = int(slurm_memory.convert_to("m").value)
+        memory_mb = buckets[0].memory.convert_to("m").value 
+        if dampen_memory is not None:
```

Comment:
I am not seeing dampen_memory being set anywhere. It defined as `None` above in the diff. Should we be comparing `memory_mb`?

Reply:
nevermind its set below.